ci/build-tarballs.sh

```diff
@@ -14,28 +14,30 @@
 
 # where PLATFORM is e.g. x86_64-linux, aarch64-linux, ...
 
+set -euo pipefail
+
 set -ex
 
```

Comment:
I like always using `set -euo pipefail` because of http://redsymbol.net/articles/unofficial-bash-strict-mode/, what do you think about changing `set -ex` to just `set -x`?

Reply:
Sure, that makes sense!